Libraries/LibGfx/PainterSkia.cpp

```diff
@@ -177,9 +180,54 @@ static SkPaint to_skia_paint(Gfx::PaintStyle const& style)
         SkMatrix matrix;
         auto shader = SkGradientShader::MakeTwoPointConical(start_sk_point, start_radius, end_sk_point, end_radius, colors.data(), positions.data(), color_stops.size(), SkTileMode::kClamp, 0, &matrix);
         paint.setShader(shader);
-        return paint;
     }
-    return {};
+}
+
+static void apply_filters(SkPaint& paint, ReadonlySpan<Gfx::Filter> filters)
```

Comment:
instead of:
1) copy paste a chunk of Gfx::Filter -> SkColorFilters transform function to start Canvas2D implementation
2) deduplicate it later by introducing shared helper

could you rearrange commit history to be:
1) move Gfx::Filter -> SkColorFilters transform function from DisplayListPlayerSkia in a shared helper
2) use shared to implement Canvas2D filters